Dockerfile.autotune

```diff
@@ -93,4 +94,4 @@ EXPOSE 8080
 ENV JAVA_HOME=${AUTOTUNE_HOME}/app/jre \
     PATH="${AUTOTUNE_HOME}/app/jre/bin:$PATH"
 
-ENTRYPOINT bash target/bin/Autotune
+ENTRYPOINT bash target/bin/CreateKruizeDBElements & bash target/bin/Autotune
```

Comment:
This cannot work ... we cannot have two processes in a container. I suggest that we embed all of it in Autotune itself

Reply:
Updated